src/common/AndroidUtils.ts

```diff
@@ -510,45 +510,57 @@ export class AndroidUtils {
                 return AndroidUtils.emulatorHasPort(resolvedEmulator);
             })
             .then(async (port) => {
-                const resolvedPortNumer = port
+                const resolvedPortNumber = port
                     ? port
                     : await AndroidUtils.getNextAvailableAdbPort();
 
-                if (resolvedPortNumer === port) {
+                if (resolvedPortNumber === port) {
                     // already is running on a port
                     const isWritable = await AndroidUtils.isEmulatorSystemWritable(
-                        resolvedPortNumer
+                        resolvedPortNumber
                     );
 
                     if (writable === false || isWritable === true) {
                         // If we're not asked for a writable, or if it already
                         // is writable then we're done so just return its port.
-                        return Promise.resolve(resolvedPortNumer);
+                        return Promise.resolve(resolvedPortNumber);
                     }
 
                     // mismatch... shut it down and relaunch it in the right mode
-                    await AndroidUtils.stopEmulator(resolvedPortNumer);
+                    CommonUtils.updateCliAction(
+                        'emulator currently launched without the -writable-system flag so shutting it down first'
```

Comment:
Minor: I propose a little more concise for a status message:

> Emulator currently launched without -writable-system flag. Shutting down.

Reply:
Also, can we start all of these messages with capital letters? We're a little inconsistent with our format, across repos, but if we're going to standardize (which I think we should), I'd prefer they start like a sentence.

Reply:
Sure I can do that. The reason I didn't is b/c the format would be `Action Message...Status Message` for example:

```
Starting device... emulator currently launched without -writable-system. Shutting down.
Starting device... relaunching sfdxdebug on port 5572 with -writable-system
Starting device... waiting for sfdxdebug to boot
Starting device... done
```

The last one (`done`) is automatically printed in lowercase when you call `cli.action.stop` so because of that I kept all of these as lowercase too.

Reply:
Ah yes I forgot about the overall message format too. Yeah, that's fine then.